### PR TITLE
fix(api-graphql): events ws connect sig

### PR DESCRIPTION
**Pull request by iartemiev**
#### Description of changes

Sigv4 calculation for Event API WS Connect was incorrectly appending a `/connect` path to the request URL. This is required for GQL Realtime WS Connect, but breaks for Event API. 

#### Issue #, if available

https://github.com/aws-amplify/amplify-js/issues/14001

#### Description of how you validated changes
* Manual - events & graphql realtime WS connect & sub with IAM/IdP
* E2Es: https://github.com/aws-amplify/amplify-js/actions/runs/11804564727

#### Checklist

- [x] PR description included
- [x] `yarn test` passes
- [x] Unit Tests are [changed or added](https://github.com/aws-amplify/amplify-js/blob/main/CONTRIBUTING.md#steps-towards-contributions)
- [x] Relevant documentation is changed or added (and PR referenced)

### CODE REVIEW

File: packages/api-graphql/src/Providers/AWSAppSyncEventsProvider/index.ts [hunk 1]
```diff
@@ -90,7 +95,7 @@ export class AWSAppSyncEventProvider extends AWSWebSocketProvider {
 			appSyncGraphqlEndpoint,
 			authenticationType,
 			query,
-			variables,
+			// variables,
```
Comment (stocaaro) [inline packages/api-graphql/src/Providers/AWSAppSyncEventsProvider/index.ts:98]: Did you mean to leave this commented out?

Comment (iartemiev) [reply packages/api-graphql/src/Providers/AWSAppSyncEventsProvider/index.ts:98]: Yes. We'll need that down the road when we enable WS publish

Comment (stocaaro) [reply packages/api-graphql/src/Providers/AWSAppSyncEventsProvider/index.ts:98]: Maybe delete it for now? Commented like this makes it look like a mistake.

Comment (iartemiev) [reply packages/api-graphql/src/Providers/AWSAppSyncEventsProvider/index.ts]: Updated

**Review by stocaaro — APPROVED** [review]

**Review by atierian — APPROVED** [review]

**Pull request merged**